specs/0119-gamification-badges.md

```diff
@@ -48,26 +48,37 @@ The badge system consists of various badges awarded based on user activities, su
+# Technical Specification
 
-# Technical specification
-
-The badge system will be integrated into the existing Pontoon interface, with badges displayed on user profiles. Status icons will be also be visible alongside their avatars in comments and suggestions. The system will track user activities and automatically award badges based on predefined criteria. Detailed implementation steps include:
-
-1. **Backend Development:**
-   - Track user activities such as translations submitted, translations reviewed, and promotions.
-   - Store badge levels and criteria.
-   - Implement logic to award badges based on tracked activities.
+The badge system will be integrated into the existing Pontoon interface, with badges displayed on user profiles. Status icons will also be visible alongside user avatars in comments and suggestions. The system will track user activities and automatically award badges based on predefined criteria. Detailed implementation steps include:
 
+1. **Backend Development Criteria:**
+   - **Activity Tracking:** Track user activities such as translations submitted, translations reviewed, and promotions.
+   - **Award Logic:** Implement logic to award badges based on tracked activities.
+   - **Data Consideration:** Only new data will be considered for badges (i.e., the data will be tracked and counted towards the badges from the time the feature goes live).
+   - **Badge Levels:** Store badge levels and criteria.
+   - **Badge Replacement:** When a user reaches a new badge level, it will replace the older version of the badge.
+   - **Notifications:** Send notifications to users to communicate their badge achievements (e.g., advancing to a new badge level or obtaining a badge).
+   
 2. **Frontend Development:**
-   - Display badges on user profiles.
-   - Show status icons universally.
+   - **Badge Display:** Display badges on user profiles under an "Achievements" section. 
+     - Example: ![Achievements Section](0119/achievements-section.png)
+   - **Status Icons:** Show status icons universally in comments and suggestions.
+     - Example: ![Suggestions](0119/suggestions.png)
+     - Example: ![Comments](0119/comments.png)
+     - **Note:** The exact design of the status icons is not finalized yet. The circle on the user icon represents a placeholder.
+   - **New User Tag:** Within the user profile, display a "New User" tag if the new user criteria are met.
+     - Example: ![New User Tag](0119/new-user-tag.png)
+     - **Note:** Similar tags already exist for team managers, translators, and administrators and are already displayed in the user profile.
 
 # Roles
 **Impacts:**
 - **Contributors and Translators:** Motivated to submit and review translations.
 - **Locale Managers:** Encouraged to mentor and promote contributors.
 
 # Mockups
-- To be included in a future version.
\ No newline at end of file
+- To be included in a future version.
```

Comment:
Mockups are now actually included... But I would move them to this section and potentially reference them from the main text, which is currently a bit more difficult to read.